server/repository/src/db_diesel/assets/asset_catalogue_item_row.rs

```diff
@@ -74,16 +76,6 @@ impl<'a> AssetCatalogueItemRowRepository<'a> {
         Ok(())
     }
 
-    pub fn insert_one(
```

Comment:
Deleted this as it's not being used and makes the code simplier if we always use upsert with Sync

Reply:
🤔  can you soft delete this?